src/main/ipv-stub/utils/user-identity-example-data.ts

```diff
@@ -0,0 +1,76 @@
+import { UserIdentity } from "../types";
```

Comment:
maybe this should live in data dir?

Reply:
Noted - something to look back on as there are a few tidy ups around structure  and naming.